test/astar.cc

```diff
@@ -1632,6 +1632,50 @@ TEST(ComplexRestriction, WalkVias) {
   }
 }
 
+TEST(Astar, BiDirTrivial) {
```

Comment:
while reviewing the two branches in bidirectional a* it was obvious that the second branch, though it should be never called other than in this test, was also buggy